### PR TITLE
bayesCor optionNames - NOT YET TESTED

### PR DESCRIPTION
**Pull request by AlexanderLyNL**
Changed option names for Simon. 

fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/1940

### CODE REVIEW

**Comment by AlexanderLyNL** [comment]
So after updating the packages as recommended by Don, I get the same error message as on GitHub actions. 

Note that I've edited `CorrelationBayesian` but not `Correlation`. The unit test fails oddly in the sense that browser (`jaspResults[["mainTable"]]$print()` and `jaspResults[["mainTable"]]$printHtml()` gives me something else than the html. 

```
results <- jaspTools::runAnalysis("Correlation", "debug.csv", options)

## browser() statement before return()

Browse[2]> jaspResults[["mainTable"]]$print()
table Correlation Table:
	status: running
	                                                              
	                                 contNormal    contGamma   
	                               |-------------|------------|
	Variable var1             | contNormal  | contGamma  |
	                               |-------------|------------|
	              n                | —         |            |
	                               |-------------|------------|
	              Pearson's r      | —         |            |
	                               |-------------|------------|
	              p-value          | —         |            |
	                               |-------------|------------|
	              VS-MPR           | —         |            |
	                               |-------------|------------|
	              Upper 95% CI     | —         |            |
	                               |-------------|------------|
	              Lower 95% CI     | —         |            |
	                               |-------------|------------|
	              Spearman's rho   | —         |            |
	                               |-------------|------------|
	              p-value          | —         |            |
	                               |-------------|------------|
	              VS-MPR           | —         |            |
	                               |-------------|------------|
	              Upper 95% CI     | —         |            |
	                               |-------------|------------|
	              Lower 95% CI     | —         |            |
	                               |-------------|------------|
	              Kendall's Tau B  | —         |            |
	                               |-------------|------------|
	              p-value          | —         |            |
	                               |-------------|------------|
	              VS-MPR           | —         |            |
	                               |-------------|------------|
	              Upper 95% CI     | —         |            |
	                               |-------------|------------|
	              Lower 95% CI     | —         |            |
	                               |-------------|------------|
	              n                | 100         | —        |
	                               |-------------|------------|
	              Pearson's r      | -5.9200e-02 | —        |
	                               |-------------|------------|
	              p-value          | 0.558       | —        |
	                               |-------------|------------|
	              VS-MPR           | 1.0000e+00  | —        |
	                               |-------------|------------|
	              Upper 95% CI     | 1.3883e-01  | —        |
	                               |-------------|------------|
	              Lower 95% CI     | -2.5268e-01 | —        |
	                               |-------------|------------|
	              Spearman's rho   | -3.4179e-02 | —        |
	                               |-------------|------------|
	              p-value          | 0.735       | —        |
	                               |-------------|------------|
	              VS-MPR           | 1.0000e+00  | —        |
	                               |-------------|------------|
	              Upper 95% CI     | 1.6334e-01  | —        |
	                               |-------------|------------|
	              Lower 95% CI     | -2.2906e-01 | —        |
	                               |-------------|------------|
	              Kendall's Tau B  | -2.6667e-02 | —        |
	                               |-------------|------------|
	              p-value          | 0.694       | —        |
	                               |-------------|------------|
	              VS-MPR           | 1.0000e+00  | —        |
	                               |-------------|------------|
	              Upper 95% CI     | 9.7081e-02  | —        |
	                               |-------------|------------|
	              Lower 95% CI     | -1.5041e-01 | —        |
	                               |-------------|------------|
```

but I get the following output: 

Screenshot 2022-08-17 at 17 08 40

File: inst/qml/CorrelationBayesian.qml [hunk 1]
```diff
@@ -42,11 +42,11 @@ Form
...
 		title: qsTr("Additional Options")
-		CheckBox {	name: "displayPairwise";		label: qsTr("Display pairwise table")					}
-		CheckBox {	name: "reportBayesFactors";		label: qsTr("Report Bayes factors"); checked: true		}
-		CheckBox {	name: "flagSupported";			label: qsTr("Flag supported correlations")				}
-		CheckBox {	name: "reportN";				label: qsTr("Sample size")								}
-		CheckBox {	name: "posteriorMedian";		label: qsTr("Posterior median"); debug: true			}
+        CheckBox {	name: "pairwiseDisplay";                label: qsTr("Display pairwise table")					}
+        CheckBox {	name: "bayesFactorReport";              label: qsTr("Report Bayes factors"); checked: true		}
+        CheckBox {	name: "supportCorrelationFlagged";		label: qsTr("Flag supported correlations")				}
+        CheckBox {	name: "sampleSizeReport";               label: qsTr("Sample size")								}
```
Comment (Kucharssim) [inline inst/qml/CorrelationBayesian.qml:48]: should this be just `sampleSize` for consistency with classical corrs?

File: R/correlationbayesian.R [hunk 2]
```diff
@@ -86,20 +86,27 @@ CorrelationBayesian <- function(jaspResults, dataset=NULL, options, ...) {
         v1 <- NA
         v2 <- NA
       } else {
-        v1 <- dataset[[.v(var1)]]
-        v2 <- dataset[[.v(var2)]]
+        v1 <- dataset[[jaspBase::encodeColNames(var1)]]
+        v2 <- dataset[[jaspBase::encodeColNames(var2)]]
```
Comment (Kucharssim) [inline R/correlationbayesian.R:90]: are you sure you need the encoding at all? assuming `var1` and `var2` come from an options list, they should be already encoded by JASP.

File: inst/Upgrades.qml [hunk 3]
```diff
@@ -174,4 +174,77 @@ Upgrades
...
+        ChangeRename { from: "reportN";								to: "sampleSizeReport"						}
+
+        ChangeJS
+        {
+            name:		"alternative"
+            jsFunction:	function(options)
+            {
+                switch(options["alternative"])
+                {
+                    case "two.sided":				return "twoSided"	;
```
Comment (Kucharssim) [inline inst/Upgrades.qml:197]: here the upgrader fails, as you need to put in the default option:

```suggestion
                    case "two.sided":				return "twoSided"	;
                    default:						return options["alternative"];
```

File: inst/Upgrades.qml [hunk 4]
```diff
@@ -174,4 +174,77 @@ Upgrades
 		
 
 	}
+
+    Upgrade
```
Comment (Kucharssim) [inline inst/Upgrades.qml:178]: the formatting in this upgrade is inconsistent (replace spaces with tabs)

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks good, just a couple of suggestions. Also, please rename `ciValue` -> `ciLevel`

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**